charts/probate-persistence-service/values.preview.template.yaml

```diff
@@ -0,0 +1,16 @@
+java:
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+  environment:
+    LIQUIBASE_AT_STARTUP: true
+    PROBATE_PERSISTENCE_SHOW_SQL: true
+
+    PROBATE_POSTGRESQL_HOSTNAME: "{{ .Release.Name }}-postgresql"
+    PROBATE_POSTGRESQL_DATABASE: "{{ .Values.postgresql.postgresqlDatabase}}"
+    PROBATE_POSTGRESQL_USER: "{{ .Values.postgresql.postgresqlUsername}}"
+    PROBATE_POSTGRESQL_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+  postgresql:
+    enabled: true
+  keyVaults:
```

Comment:
i prefer this to avoid empty errors : https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.preview.template.yaml#L12

Reply:
ha, didn't know about that